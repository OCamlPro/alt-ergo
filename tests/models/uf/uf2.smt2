(set-logic ALL)
(set-option :produce-models true)
(declare-fun f (Int) Int)
(declare-const a Int)
(declare-const b Int)
(assert (distinct (f a) (f b)))
(check-sat)
(get-model)
