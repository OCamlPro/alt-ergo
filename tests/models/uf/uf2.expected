
unknown
(
  
 ; Functions

(define-fun f ((arg_0 Int)) Int (ite (= arg_0 a) 2 0))

 ; Constants

(define-fun a () Int 0)

(define-fun b () Int (- 2))

 ; Arrays not yet supported


)
